class PatientNotFound(Exception):
    pass

class UnexpectedError(Exception):
    pass